Description: Disable connection pool tests
 Debian specific patch
Author: Pirate Praveen <praveen@debian.org>
Last-Updated: 2014-05-04

--- a/spec/connection_pool_spec.rb
+++ b/spec/connection_pool_spec.rb
@@ -5,7 +5,7 @@ QUERY = "select sleep(#{DELAY})"
 
 describe EventMachine::Synchrony::ConnectionPool do
 
-  it "should queue requests if pool size is exceeded" do
+  xit "should queue requests if pool size is exceeded" do
     EventMachine.run do
 
       db = EventMachine::Synchrony::ConnectionPool.new(size: 1) do
@@ -29,7 +29,7 @@ describe EventMachine::Synchrony::Connec
     end
   end
 
-  it "should execute multiple async pool requests within same fiber" do
+  xit "should execute multiple async pool requests within same fiber" do
     EventMachine.run do
 
       db = EventMachine::Synchrony::ConnectionPool.new(size: 2) do
@@ -53,7 +53,7 @@ describe EventMachine::Synchrony::Connec
     end
   end
 
-  it "should share connection pool between different fibers" do
+  xit "should share connection pool between different fibers" do
     EventMachine.run do
 
       db = EventMachine::Synchrony::ConnectionPool.new(size: 2) do
@@ -84,7 +84,7 @@ describe EventMachine::Synchrony::Connec
     end
   end
 
-  it "should share connection pool between different fibers & async requests" do
+  xit "should share connection pool between different fibers & async requests" do
     EventMachine.run do
 
       db = EventMachine::Synchrony::ConnectionPool.new(size: 5) do
